File: app/src/main/java/com/agotakiss/androidtest/data/models/LoadMoviesResponse.java
package com.agotakiss.androidtest.data.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class LoadMoviesResponse implements Serializable {

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<MovieDataModel> movies;

    @SerializedName("total_results")
    private int totalResults;

    @SerializedName("total_pages")
    private int totalPages;

    public int getPage() {
        return page;
    }

    public List<MovieDataModel> getMovies() {
        return movies;
    }

    public int getTotalResults() {
        return totalResults;
    }

    public int getTotalPages() {
        return totalPages;
    }
}

File: app/src/main/java/com/agotakiss/androidtest/data/models/LoadMoviesResponse.kt
package com.agotakiss.androidtest.data.models

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class LoadMoviesResponse : Serializable {

    @SerializedName("page")
    val page: Int = 0

    @SerializedName("results")
    val movies: List<MovieApiModel>? = null

    @SerializedName("total_results")
    val totalResults: Int = 0

    @SerializedName("total_pages")
    val totalPages: Int = 0
}

